apps/vr-tests/src/stories/CoachmarkNext.stories.tsx

```diff
@@ -0,0 +1,71 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
```

Comment:
could you duplicate the this test for Coachmark in OUFR as well? similar to other tests we have. so we always have 2 results side by side